### PR TITLE
added required files for password leak checker

### LINKED ISSUE #129: Password Strength Checker
### Describe the new script you want
Basically this script will tell how many times the password has been breached before in data breaches so you can estimate the strength of a password

### Describe the solution you'd like us to follow
I am using an API for fetching of haveibeenpwned.com to bring the data, and make sure you don't have to enter your password directly on the site for more security.

## Do you want to work on this?
- [x] yes
- [ ] no

### PR DESCRIPTION
**Pull request by rex2828**
# Password Leak Checker

This python script will allow you to check if your password has been compromised in any other data breaches.
This is the simple way of checking how strong is your password.
Fixes #129 

## Type of change

Please delete options that are not relevant.
- [x] New Script (non-breaking change which adds a new script to the repository)

## Has This Been Tested?

- [x] I confirm that I have tested my script locally and verified that it works as intended.

## Checklist:

- [x] I have gone through the contributing guidelines before making this Pull Request.
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: scripts/PasswordStrengthChecker/script.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+# this way you don't need to send your full password and can get the appropriate data locally
+# that is why this script is more secure than manually going to the website and typing out your password their
+def request_api_data(query_char):
+    url = "https://api.pwnedpasswords.com/range/" + query_char
+    res = requests.get(url)
+    if (
+        res.status_code != 200
+    ):  # if res.status_code is not 200 then there is some problem
+        raise RuntimeError(
+            f"Error fetching: {res.status_code}, chech the API and try again."
```
Comment (OBITORASU) [inline scripts/PasswordStrengthChecker/script.py:18]: there is a typo, `chech` should be `check`

**Review by OBITORASU — CHANGES_REQUESTED** [review]

**Comment by OBITORASU** [comment]
LGTM! That aside, I think we should change the title of this script a bit, instead of password strength checker, lets call this one something like `a python script to check if your password has been leaked in a data breach`. How does that sound?

**Comment by OBITORASU** [comment]
The title can be converted to `Password leak checker`.

**Comment by rex2828** [comment]
Okay I will change the title @OBITORASU

**Comment by OBITORASU** [comment]
Please make the same changes in the documentation (READMEs) as well!

**Comment by OBITORASU** [comment]
Thank you for contributing.

**Pull request merged**